model_signing/signature/signature.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This package provides the functionality to sign models"""
+import abc
+
+from in_toto_attestation.v1 import statement
+from sigstore_protobuf_specs.dev.sigstore.bundle import v1 as bundle_pb
+
+
+class VerificationMaterial(metaclass=abc.ABCMeta):
+    """Generic verification material"""
+
+
+class SigstoreVerificationMaterial(VerificationMaterial):
```

Comment:
Can this be just a subclass of `signing.Signature, which will just wrap the bundle?